exchanges/account/account.go

```diff
@@ -268,22 +279,31 @@ func (s *Service) Update(incoming *Holdings, creds *Credentials) error {
 
 // load checks to see if there is a change from incoming balance, if there is a
 // change it will change then alert external routines.
-func (b *ProtectedBalance) load(change Balance) {
+func (b *ProtectedBalance) load(change *Balance) error {
+	if change == nil {
+		return fmt.Errorf("%w for '%T'", common.ErrNilPointer, change)
+	}
 	b.m.Lock()
 	defer b.m.Unlock()
+	if !b.updatedAt.IsZero() && !b.updatedAt.Before(change.UpdatedAt) {
```

Comment:
Now that the time is assigned in Update method. Could we please force storage, so if `!b.updatedAt.IsZero()` we then return an error. 🙏 And update the test.